vignettes/NSSP_nowcast.Rmd

```diff
@@ -0,0 +1,510 @@
+---
+title: "Nowcasting U.S. National Syndromic Surveillance Program (NSSP) data"
+description: "A nowcasting example applied to data from NSSP"
```

Comment:
as in the news suggest refocus sss data using .,... (NSSP). 

Need to change title of the Rmd to use lower case nssp

Reply:
news suggest refocus sss ?